app/helpers/uploads_helper.rb

```diff
@@ -54,5 +54,61 @@ def match_errors(upload)
       return error_map, error_attributes
     end
 
+    def should_switch_highlight?(data_key, pop_key, specifics, rationale)
```

Comment:
Please run rubocop on this file. You only need to worry about the lines you added.